install_config/aggregate_logging.adoc

```diff
@@ -196,6 +205,10 @@ when `openshift_logging_use_ops` is set to `true`.
 |Equivalent to `openshift_logging_curator_memory_limit` for Ops cluster
 when `openshift_logging_use_ops` is set to `true`.
 
+|`openshift_logging_kibana_image`
+|The image version for Kibana.
+For example: `openshift_logging_kibana_image=<registry_url>:5000/openshift3/logging-kibana:v3.X.X`
```

Comment:
Shall we use offical images as an example?   registry.redhat.io/openshift3/ose-logging-kibana5:v3.11.  it is kibana5 in 3.11